db/migrate/20160412223500_add_institution_name_full_to_site.rb

```diff
@@ -0,0 +1,5 @@
+class AddInstitutionNameFullToSite < ActiveRecord::Migration
+  def change
+    add_column :sites, :institution_name_full, :string
```

Comment:
Before merging, we'll probably want to collapse these two migrations for our future sanity.